api/routes.py

```diff
@@ -1,15 +1,21 @@
 # app.py -- routes for the app
 import json
 import logging
-from hmac import new as hmac_new
+import hmac
 from datetime import datetime
 
-from flask import request, make_response, g
+from flask import request, make_response
 
 from api import app
 from api.queue import Queue
 from api.payload import Payload
 
+try:
+    from api.secrets import TOKENS
+except ModuleNotFoundError:
+    logging.error('Required secrets file not found. See api/secrets.example.py for setup information.')
+    import sys
+    sys.exit(1)
```

Comment:
Any reason you prefer to exit manually instead of raising the error directly?

Reply:
oops! i didn't mean to commit that, it was just a placeholder until i decided the best way to do it.

really all i wanted was to log some kind of useful error message. should i just 
`raise Exception('Required secrets file not found. See api/secrets.example.py for setup information.')`?

Reply:
Basically yeah! I would `raise ModuleNotFoundError('Required secrets file not found. See api/secrets.example.py for setup information.')` just to be as specific as possible, but that's more of a style point.

Reply:
I forgot you can just use the bare `raise` keyword to raise the exception from context. This way we add the extra info before the traceback, and send it through `logging` in case we ever implement custom logging